docs/how-to/deploy_sdcore_upf_machine.md

```diff
@@ -0,0 +1,40 @@
+# Deploy SD-Core User Plane Machine
+
+This guide covers how to deploy the User Plane Function (UPF) as a machine charm.
+
+## Requirements
+
+- A host with a CPU supporting AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
+- [Juju][Juju] controller bootstrapped to a LXD cluster
+- A Juju machine manually representing the host added to the controller
```

Comment:
I find this sentence confusing, what do you think of this? 
```suggestion
- A machine added to the Juju controller
```

Reply:
Looks better, thank you.